pkg/config/config.go

```diff
@@ -146,6 +146,11 @@ func (vm *VirtualMachine) AddDevice(dev VirtioDevice) {
 	vm.Devices = append(vm.Devices, dev)
 }
 
+// AddDevices adds a list of devices to vm.
```

Comment:
This is what `AddDevice` should have been ;) maybe change it to `return AddDevices(dev)`

Reply:
Good idea!